packages/ovm/test/contracts/execution-manager.call-opcodes.spec.ts

```diff
@@ -415,30 +423,38 @@ describe('Execution Manager -- Call opcodes', () => {
       })
     })
 
-    it('fails on ovmSTATICCALL to CREATE', async () => {
+    it('Fails to create on ovmSTATICCALL to CREATE', async () => {
       const data: string = `${executeCallToCallContractData}${staticCallMethodId}${callContract2Address32}${createMethodIdAndData}`
 
-      await TestUtils.assertThrowsAsync(async () => {
-        // Note: Send transaction vs call so it is persisted
-        await wallet.sendTransaction({
-          to: executionManager.address,
-          data,
-          gasLimit: 6_700_000,
-        })
+      // Note: Send transaction vs call so it is persisted
+      const receipt = await wallet.sendTransaction({
+        to: executionManager.address,
+        data,
+        gasLimit: 6_700_000,
       })
+
+      const creatSucceeded = await didCreateSucceed(
+        executionManager,
+        receipt.hash
+      )
+      creatSucceeded.should.equal(false, 'Create should have failed!')
     })
 
     it('fails on ovmSTATICCALL to CREATE2', async () => {
       const data: string = `${executeCallToCallContractData}${staticCallMethodId}${callContract2Address32}${create2MethodIdAndData}`
 
-      await TestUtils.assertThrowsAsync(async () => {
-        // Note: Send transaction vs call so it is persisted
-        await wallet.sendTransaction({
-          to: executionManager.address,
-          data,
-          gasLimit: 6_700_000,
-        })
+      // Note: Send transaction vs call so it is persisted
+      const receipt = await wallet.sendTransaction({
+        to: executionManager.address,
+        data,
+        gasLimit: 6_700_000,
```

Comment:
Doesn't have to be in this PR because this is elsewhere but can we set this as the default somewhere?